contracts/OgvStaking.sol

```diff
@@ -33,6 +33,11 @@ contract OgvStaking is ERC20Votes {
     mapping(address => uint256) public rewardDebtPerShare;
     uint256 public accRewardPerShare; // As of the start of the block
 
+    // Used to track any manual `delegate()` method call. When this isn't
+    // set to true, voting powers are delegated to the receiver of the stake
+    // when `stake()` or `extend()` method are called.
+    mapping(address => bool) public hasManualDelegation;
```

Comment:
"Manual" may be the wrong word in this variable name, since almost everyone with this set will have had it automatically set by via this new code?

Reply:
Resolved in PR #377